### DISCUSSION TITLE
Can you do stereo with out side DAC side board?

### DISCUSSION THREAD

**Discussion by RaymondDay61** [frntc/SIDKick-pico#45] [General]
Can you do stereo with the the side board on it?

Or the only way to get the 2nd channel is with the side board?

I been playing music with it in 3 voice mono and it sound just like a real SID.

But love to have Stereo. Were on the board do you solder the 2nd channel wire?

**Comment by frntc** [comment]
With the current PCB-revision you can only get stereo with the external DAC module (but of course you can play dual-SID in mono).

For the next PCB-revision there will be a variant with the DAC build-in, it can output stereo without any additional components.

**Reply by RaymondDay61** [reply]
I had it working with out the DAC board at 1st and it sounded real good. Can't they do the same for the 2nd channel have a pin out for that on the board? So you would not have to use the DAC for true stereo. Not just all out one speaker like the default is.

This is very good. Just wish it had a pin for stereo with out the DAC.

**Comment by RaymondDay61** [comment]
Thank you I hooked it up in stereo both the hard phone and out the video port on pin 7 work. I solder a wire on the DAC board from "ROUT" to pin 7 I guess the right sound is the 2nd channel. It does not say that in here that I would find.

When I turn a dead test it says the SID is bad. It plays and sound good but I think it should make a nose sound too but I don't hear that. It my be I have a old dead test.

-Raymond Day

**Reply by frntc** [reply]
The typical resistors used in dead test dongles might produce potentiometer values which are a bit too low/high for the testing threshold. That's because of the level shifting involved, but paddles work nonetheless.
